framework/src/modules/chain/dpos/account.js

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const BigNum = require('@liskhq/bignum');
+
+const _mergeRewardsAndDelegates = (delegatePublicKeys, rewards) =>
+	delegatePublicKeys
+		.map((publicKey, index) => ({
+			publicKey,
+			reward: new BigNum(rewards[index]),
+			blocksForged: 1,
+			isGettingRemainingFees: index === delegatePublicKeys.length - 1,
+		}))
+		.reduce((acc, curr) => {
+			const delegate = acc.find(
+				({ publicKey }) => publicKey === curr.publicKey,
+			);
+
+			if (!delegate) {
```

Comment:
This will always be `true` if no delegate miss a block in that round.
If any delegate miss a block in the round, another delegate will forge twice and their reward needs to be updated

Reply:
Got it!